src/components/PostItem/PostItem.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC, useMemo } from 'react';
+import Card from '@mui/material/Card';
+import CardHeader from '@mui/material/CardHeader';
+import { Checkbox, IconButton } from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import CardActions from '@mui/material/CardActions';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { IPost } from '../../models/IPosts';
+
+type TPostItemProps = {
+  post: IPost;
+  remove: (id: number) => void;
+  update: (post: IPost) => void;
```

Comment:
❗️
Это пропы-коллбэки, их принято называть так
```suggestion
  onDelete: (id: number) => void;
  onUpdate: (post: IPost) => void;
```

Reply:
готово! on - добавил